packages/kolibri-tools/lib/lint.js

```diff
@@ -191,9 +191,8 @@ function lint({ file, write, encoding = 'utf-8', silent = false } = {}) {
               if (linted.trim() !== (stylinted || code).trim()) {
                 notSoPretty = true;
               }
-              if (linted.trim() !== code.trim()) {
-                styleCodeUpdates.push(() => callback(linted));
```

Comment:
What was the issue you were seeing here?

Reply:
`insertContent` (the function responsible for inserting a block while adjusting it's indentation and content spacing) is passed here as `callback` function when styles are being processed. This was working properly only when there were some stylelint updates of the block. When there were none, this condition caused that the correct indentation wasn't applied to style block, for example:

```
<style lang="scss" scoped>
.red {
  color: red;
}
</style>
```

would not be formatted to
```
<style lang="scss" scoped>

  .red {
    color: red;
  }

</style>
```

In such cases, HTMLHint saved it and reported `"'Top-level content should be surrounded by one empty line.'"` and `"Top-level content should be indented two spaces."` so you could still fix it manually. But I removed those two checks from HTMLHint in this PR so I also reviewed existing auto-formatting functionality and adjusted it to work for those scenarios, too.